testsuite/testscripts/test-zeppelin.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import os
+import urllib2
+
+master_url = "http://" + os.environ['ZEPPELIN_MASTER_NODE'] + ":" + os.environ['ZEPPELIN_SERVER_PORT']
+
+response = urllib2.urlopen(master_url).getcode()
+
+if response == 200:
+    print "Success"
+    return 0
```

Comment:
See my upcoming comment on test-validation, but perhaps these print statements will need to be enhanced.